scripts/speak.py

```diff
@@ -3,11 +3,12 @@
 import requests
 from config import Config
 cfg = Config()
-import gtts
 
+# Remove the import of gtts
+# import gtts
 
-# TODO: Nicer names for these ids
-voices = ["ErXwobaYiN019PkySvjV", "EXAVITQu4vr4xnSDxMaL"]
+# Change voices to macOS voice identifiers
+voices = ["com.apple.speech.synthesis.voice.siri_female", "com.apple.speech.synthesis.voice.siri_male"]
```

Comment:
These voices are for elevenlabs.io, not GTTS.

Reply:
He also removed Windows support by hardcoding to macos below and not making it configurable.

Reply:
My bad, I didn't mean to cut out windows users. 
My bot seems to have only pushed the first commit. 
will fix this in a few hours